Source/missiles.cpp

```diff
@@ -1125,17 +1122,14 @@ bool PlayerMHit(int pnum, MonsterStruct *monster, int dist, int mind, int maxd,
 		hper += (monster->mLevel * 2) - (player._pLevel * 2) - (dist * 2);
 	}
 
-	if (hper < 10)
-		hper = 10;
-	if (currlevel == 14 && hper < 20) {
-		hper = 20;
-	}
-	if (currlevel == 15 && hper < 25) {
-		hper = 25;
-	}
-	if (currlevel == 16 && hper < 30) {
-		hper = 30;
-	}
+	int minhit = 10;
+	if (currlevel == 14)
+		minhit = 20;
+	if (currlevel == 15)
+		minhit = 25;
+	if (currlevel == 16)
+		minhit = 30;
+	hper = clamp(hper, minhit, 100);

```

Comment:
are we sure it's ok to cap this at 100?
```suggestion
	hper = std::max(hper, minhit);
```